pyeudiw/satosa/frontends/openid4vci/endpoints/credential_endpoint.py

```diff
@@ -38,21 +38,21 @@ def validate_request(self, context: Context, entity: OpenId4VCIEntity) -> OpenId
 
         Args:
             context (Context): The SATOSA context containing request data.
-            entity (OpenId4VCIEntity): The stored session/entity related to the request.
+            entity (dict): The stored session/entity related to the request.
 
         Raises:
             pydantic.ValidationError: If the request body does not match the expected schema.
         """
         c_req = CredentialEndpointRequest.model_validate(self._get_body(context), context = {
-            AUTHORIZATION_DETAILS_CTX: entity.authorization_details
+            AUTHORIZATION_DETAILS_CTX: entity["authorization_details"]
```

Comment:
what about a possible KeyError exception on authorization_details?

I believe that it should be handled accordingly